src/main/java/co/inventorsoft/academy/springBootTask/repository/impl/TeamRepositoryImpl.java

```diff
@@ -0,0 +1,90 @@
+package co.inventorsoft.academy.springBootTask.repository.impl;
+
+import co.inventorsoft.academy.springBootTask.domain.entity.Team;
+import co.inventorsoft.academy.springBootTask.repository.EntityMapper;
+import co.inventorsoft.academy.springBootTask.repository.TeamRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Repository;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+@RequiredArgsConstructor
+@Slf4j
+public class TeamRepositoryImpl implements TeamRepository {
+
+    private static final String SQL__FIND_ALL_TEAMS = "SELECT * FROM team";
+    private static final String SQL__ADD_TEAM = "INSERT into `team`(name, `capitan`, coach) " + "values(?, ?, ?)";
+
+    @Override
+    public List<Team> findAll() {
+        List<Team> teamList = new ArrayList<Team>();
+        PreparedStatement pstmt = null;
+        ResultSet rs = null;
+        Connection con = null;
+        try {
+            con = DBManager.getInstance().getConnection();
```

Comment:
`Connection`, `PreparedStatement` and `ResultSet` should be closed when usage is complete.
`java.sql.Connection#close`